root.tf

```diff
@@ -484,6 +493,36 @@ module "export_step_function" {
   })
 }
 
+module "new_export_bucket" {
```

Comment:
Maybe rename the module something a bit more meaningful such as: `flat_format_export_bucket`